### PR TITLE
[REF] web_tour: clarify position in tour API

### PR DESCRIPTION
**Pull request by pipu-odoo**
In order to clarify and simplify the tours API, it was decided to rename
position to tooltipPosition. Indeed, the position key can be confusing
while the name tooltipPosition makes things very clear.

task~3974087

https://github.com/odoo/enterprise/pull/67352

### CODE REVIEW

File: addons/mass_mailing/static/tests/tours/mass_mailing_code_view.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ registry.category("web_tour.tours").add('mass_mailing_code_view_tour', {
             content: 'Drag the "Title" snippet from the design panel and drop it in the editor',
             async run(helpers) {
                 helpers.drag_and_drop(`[name="body_arch"] :iframe .o_editable`, {
-                    position: {
+                    tooltipPosition: {
```
Comment (pipu-odoo) [inline addons/mass_mailing/static/tests/tours/mass_mailing_code_view.js:60]: nope

File: addons/pos_restaurant/static/tests/tours/floor_screen_tour.js [hunk 2]
```diff
@@ -178,7 +178,7 @@ registry.category("web_tour.tours").add("FloorScreenTour", {
                 trigger: FloorScreen.table({ name: "5" }).trigger,
                 async run(helpers) {
                     await helpers.drag_and_drop(`div.floor-map`, {
-                        position: {
+                        tooltipPosition: {
```
Comment (pipu-odoo) [inline addons/pos_restaurant/static/tests/tours/floor_screen_tour.js:181]: no

File: addons/website/static/src/js/tours/tour_utils.js [hunk 3]
```diff
@@ -312,7 +312,7 @@ function dragNDrop(snippet, position = "bottom") {
             content: markup(_t("Click on the <b>%s</b> building block.", snippet.name)),
             trigger: `:iframe .o_snippet_preview_wrap[data-snippet-id="${snippet.id}"]`,
             noPrepend: true,
-            position: position,
+            tooltipPosition: position,
```
Comment (pipu-odoo) [inline addons/website/static/src/js/tours/tour_utils.js:315]: no

File: addons/website/static/tests/tours/edit_menus.js [hunk 4]
```diff
@@ -218,7 +218,7 @@ wTourUtils.registerWebsitePreviewTour('edit_menus', {
         trigger: '.oe_menu_editor li:contains("Contact us") .fa-bars',
         async run(helpers) {
             await helpers.drag_and_drop('.oe_menu_editor li:contains("Home")', {
-                position: {
+                tooltipPosition: {
```
Comment (pipu-odoo) [inline addons/website/static/tests/tours/edit_menus.js:221]: no

**Review by Arcasias — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**